tests/terminalApp.js

```diff
@@ -0,0 +1,6 @@
+const getIdeas = require('../src/data/ideas');
+const URL = 'https://fund-raising-project-server-side.azurewebsites.net/ideas';
```

Comment:
What do you need this URL for?

Reply:
That one left after previouse version. Fixed.